BD-2609: Update guidance re: Unsubscribe URL Tags (#6537)
braze-inc/braze-docs: _docs/_user_guide/message_building_by_channel/email/custom_email_footer.md

---
nav_title: Custom Email Footer
article_title: Custom Email Footer
page_order: 6.5
description: "This article describes how to set up a workspace-wide custom email footer."
channel:
  - email

---

# Custom email footer

> You can set a workspace-wide custom email footer which you can template into every email using the {% raw %}`{{${email_footer}}}`{% endraw %} Liquid attribute.

Reviewer: ```suggestion
> You can set a workspace-wide custom email footer which you can template into every email using the {% raw %}`{{${email_footer}}}`{% endraw %} Liquid attribute.
```

> This way, you don't have to create a new footer for every email template or email campaign you use. Changes you make to your custom footer will be reflected in all new and existing email campaigns. Remember that compliance with the [CAN-SPAM Act of 2003](https://www.ftc.gov/tips-advice/business-center/guidance/can-spam-act-compliance-guide-business) requires you to include a physical address for your company and an unsubscribe link in your emails.

Reviewer: ```suggestion
By using custom email footers, you no longer have to create a new footer for every email template or email campaign you use. Changes you make to your custom footer will be reflected in all new and existing email campaigns. Remember that compliance with the [CAN-SPAM Act of 2003](https://www.ftc.gov/tips-advice/business-center/guidance/can-spam-act-compliance-guide-business) requires you to include a physical address for your company and an unsubscribe link in your emails.
```
